### PR TITLE
Mention draft PRs in readme and fix PR documentation

### PR DESCRIPTION
**Pull request by danielgustafsson**
We do welcome draft PRs, especially now that Github has enabled it as a feature when creating PRs. Add a note on this in the main README and also fix the text regarding which branch to PR against, as we now have a supported back-branch.

Also fix a long line in the same section to make reading of the source markdown easier.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -352,12 +352,18 @@ Feel free to ask on the mailing list to learn more about the Greenplum release p
...
+repository forked from the official Greenplum and
+[send us a pull request](https://help.github.com/articles/about-pull-requests/).
+
+We welcome submissions which are work in-progress in order to get feedback early
+in the development process.  When opening the pull request, select "Draft" in
+the dropdown menu when creating the PR to clearly mark the intent of the pull
+request. Prefixing the title with "WIP:" is also good practice.
+
+All new features should be submitted against the main master branch. Bugfixes
+should too be submitted against master unless they only exist in a supported
```
Comment (berlin-ab) [inline README.md]: s/exist/exists/

File: README.md [hunk 2]
```diff
@@ -352,12 +352,18 @@ Feel free to ask on the mailing list to learn more about the Greenplum release p
...
+back-branch. If the bug exist in both master and back-branches, explain this
```
Comment (berlin-ab) [inline README.md]: s/exists/exist/

**Comment by berlin-ab** [comment]
Nice addition.

File: README.md (hunk 2 above)
Comment (danielgustafsson) [reply README.md]: Fixed, thanks

**Comment by danielgustafsson** [comment]
Pushed to master, thanks for review!

**Pull request closed without merging**